Microsoft.ML.AutoML.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28010.2050	
+# Visual Studio Version 16
```

Comment:
Undo this change. It should be able to be build on Dev 15 and Dev 16.

Reply:
K